test/verify/machineslib.py

```diff
@@ -178,13 +178,41 @@ def prepareStorageDeviceOnISCSI(m, target_iqn, orig_iqn=None):
 class TestMachines(NetworkCase):
     created_pool = False
     provider = None
+    tmp_storage = "/mnt/libvirt"
```

Comment:
/mnt is a little strange and on existing machines it may be on NFS or something. Please use /var/tmp/

Reply:
`/var/tmp` does not work. It was failing to start domain with `file system may not support O_DIRECT`. Did not give it much thought, just used the first thing than I knew would work. (Problem was when I put there storage).

Reply:
Interesting -- is /var/tmp a tmpfs on the machine you try this on? Then let's keep /mnt for now. If it gets in the way downstream, we can still easily move that to /var/lib/cockpit-test/ or so.